validator/src/cli.rs

```diff
@@ -1411,6 +1411,15 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .help("Enables the experimental accumulator hash")
                 .hidden(hidden_unless_forced()),
         )
+        .arg(
+            Arg::with_name("accounts_db_hash_threads")
```

Comment:
I've been putting all of the new args into a separate file, `validators/src/cli/thread_args.rs`. There is some "framework" in there to ensure all of the args behave consistently

One other consequence of this is that the default value is accessible at the "top level". This will be useful for down the road if we do more proper "load balancing" for various pools.